.gitignore

```diff
@@ -9,10 +9,10 @@ pkg/
 # Berkshelf
 .vagrant
 /cookbooks
-# Berksfile.lock
+Berksfile.lock
```

Comment:
Shouldn't Berksfile.lock be in .gitignore because only application cookbooks will set it?